doc-content/enterprise-only/openshift/offline-repo-proc.adoc

```diff
@@ -1,16 +1,39 @@
 [id='offline-repo-proc']
+:offline_onprem!:
+ifeval::["{context}"=="install-on-eap"]
+:offline_onprem:
+endif::[]
+ifeval::["{context}"=="install-on-jws"]
+:offline_onprem:
+endif::[]
+
 = Preparing a Maven mirror repository for offline use
 
-If your {OPENSHIFT} environment does not have outgoing access to the public Internet, you must prepare a Maven repository with a mirror of all the necessary artifacts and make this repository available to your environment.
+If your 
+ifdef::offline_onprem[]
+{PRODUCT} deployment 
+endif::offline_onprem[]
+ifndef::offline_onprem[]
+{OPENSHIFT} environment 
+endif::offline_onprem[]
+does not have outgoing access to the public Internet, you must prepare a Maven repository with a mirror of all the necessary artifacts and make this repository available to your environment.
 
-You do not need to complete this procedure if your {OPENSHIFT} environment is connected to the Internet.
+You do not need to complete this procedure if your
+ifdef::offline_onprem[]
+{PRODUCT} deployment 
+endif::offline_onprem[]
+ifndef::offline_onprem[]
+{OPENSHIFT} environment 
+endif::offline_onprem[]
+is connected to the Internet.
 
 .Prerequisites
 
 * A computer that has outgoing access to the public Internet.
```

Comment:
Prereqs must be sentences, so:
You have a computer....
or
A computer is available that has...
This is the one area that we make an exception for passive voice.